test/Riok.Mapperly.Tests/Mapping/UserMethodTest.cs

```diff
@@ -55,6 +55,29 @@ public record B(int Value);
         return TestHelper.VerifyGenerator(source);
     }
 
+    [Fact]
+    public Task InstanceMapperShouldSupportUserDefinedExistingTargetStaticMethods()
+    {
+        var source =
```

Comment:
Use `Riok.Mapperly.Tests.TestSourceBuilder.CSharp` to get correct syntax highlighting in the IDE and probable use a raw string literal.